### PR TITLE
Update script based on current Google Sheet

### PR DESCRIPTION
**Pull request by magus**
Currently the version of the script in `master` fails due to some changes in the Google Sheets document, i.e. removed or changed Sheets in the Google Sheet document.

## Changes
- Synchronize JSON `out` as of `2020-05-18`
- `README` setup instructions
- Rename `Wallpapers` to `Wallpaper` sheet reference
- Rename `Bugs` to `Insects` sheet reference
- Remove `Nook Miles` sheet reference ([#changelog](https://discordapp.com/channels/701573691426996324/701577355839406200/709618926224670763))
- `readJSON` utility function to read in JSON for aggregation into `all.json`
- `writeJSON` utility function to write in a consistent pretty JSON format

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -39,9 +39,7 @@ const ITEM_SHEETS = [
   'Art',
 ];
 
-const CREATURE_SHEETS = ['Fish', 'Bugs'];
-
-const NOOK_MILE_SHEETS = ['Nook Miles'];
```
Comment (magus) [inline src/index.ts:44]: Might be good to confirm this was actually deleted.

Comment (magus) [reply src/index.ts:44]: Here's the relevant changelog https://discordapp.com/channels/701573691426996324/701577355839406200/709618926224670763

File: README.md [hunk 2]
```diff
@@ -2,3 +2,11 @@
 
 This repository contains the code used to convert the Animal Crossing: New Horizons community items
 google [spreadsheet](https://docs.google.com/spreadsheets/d/13d_LAJPlxMa_DubPTuirkIV4DERBMXbrWQsmSh8ReK4/edit?usp=sharing) into JSON.
+
+# Setup
+
+- Visit [Google Sheets API Quickstart](https://developers.google.com/sheets/api/quickstart/nodejs)
+- Click `Enable the Google Sheets API` to download `credentials.json`
```
Comment (iamcal) [inline README.md:9]:
```suggestion
- Click `Enable the Google Sheets API` and choose `Desktop app` as your OAuth client. Download the credentials file `credentials.json`
```

File: README.md [hunk 3]
```diff
@@ -2,3 +2,11 @@
...
+- Copy `credentials.json` into the root of your local copy of this repository
+- Run `yarn start` to start a water that will automatically build into `build`
+- Run `node build` to export data from Google Sheets into local `out` folder
```
Comment (iamcal) [inline README.md:12]:
```suggestion
- Run `npm install` to install the required node modules
- Run `node build` to export data from Google Sheets into local `out` folder
```

Comment (LavaToaster) [reply README.md:12]: Don't mix and match npm/yarn, yarn is preferred here.

**Pull request merged**